pipelineUtilities.psm1

```diff
@@ -150,6 +150,7 @@ function Install-Dotnet {
         $listRuntimesOutput = dotnet --list-runtimes
         $installedDotnetRuntimes = $listRuntimesOutput | ForEach-Object { $_.Split(" ")[1] }
         Write-Host "Detected dotnet Runtimes: $($installedDotnetRuntimes -join ', ')"
+        dotnet --info
```

Comment:
Curious why this change was made?

Reply:
I forgot to remove from before; this is not needed